### DISCUSSION TITLE
Importer (CSV and JSON) plugin is not free

### DISCUSSION THREAD

**Discussion by simonschaufi** [kimai/kimai#4170] [Plugins]
### Describe the issue

https://www.kimai.org/store/keleo-importer-bundle.html says the plugin is free but it actually costs 9.99€

**Comment by kevinpapst** [comment]
It is "Pay what you want":

<img width="581" alt="Bildschirmfoto 2023-07-06 um 15 08 18" src="https://github.com/kimai/kimai/assets/533162/50ecfaba-86da-4d06-9060-9fa5c45c4a33">

Wanna be cheap?

<img width="1687" alt="Bildschirmfoto 2023-07-06 um 15 09 10" src="https://github.com/kimai/kimai/assets/533162/7a3d0d59-36e6-4e5d-831a-316d00ad4006">

Thank god, developer do not need to eat!

**Reply by simonschaufi** [reply]
ok, can you change the FREE to "PAY whatever you want"? That would be more accurate ;) Actually I didn't even think that "0" is also possible as most times you have to pay at least 1€ or so... Thanx for clarification!

> developer do not need to eat!

Of course not! They also don't need sunshine. Just a good strong Wifi and then they are happy :smile:
